docs/README_alcf832.md

```diff
@@ -0,0 +1,443 @@
+# 8.3.2 ALCF Globus Flow And Reconstruction Setup
+
+This script facilitates the transfer and processing of data files from the Advanced Light Source (ALS) at Berkeley Lab to the Argonne Leadership Computing Facility (ALCF) and the National Energy Research Scientific Computing Center (NERSC). It includes functions for transferring data using Globus and for performing tomographic reconstruction using Tomopy at ALCF.
```

Comment:
Now that we moved this into docs, the script name is a little ambigious. For this, we could use a markdown link...  [the init script](../orchestration/scritps/globaus_tomopy_flow_init.py)